engines/instance_verification/app/controllers/instance_verification/billing_check_controller.rb

```diff
@@ -14,6 +14,9 @@ def check
       is_payg = verification_provider.payg_billing_code?(iid, params[:identifier])
 
       render status: :ok, json: { flavor: is_payg ? 'PAYG' : 'BYOS' }
+    rescue InstanceVerification::Exception => e
+      logger.error "Instance metadata '#{params[:metadata]}' could not be parsed: #{e.message}"
+      render status: :unprocessable_entity, json: { flavor: 'BYOS' }
```

Comment:
Why is BYOS hardcoded here, when in line 16 there is both 'PAYG' and 'BYOS' handled?

Reply:
I suggest to add this as a comment here.